packages/react-form/src/types.ts

```diff
@@ -28,3 +28,12 @@ export type UseFieldOptions<
 > & {
   mode?: 'value' | 'array'
 }
+
+/**
+ * The return type use useForm with pre-populated generics
+ */
+export type UseFormReturnType<TForm, TFormValidator> = TFormValidator extends
```

Comment:
I would love to get rid of the TFormValidator, since TFormValidator is just Validator<TForm, unknown> | undefined. But so far my efforts have been fruitless.